.github/workflows/python-publish-package.yml

```diff
@@ -0,0 +1,67 @@
+name: Publish Python package
+on:
+  workflow_call:
+    inputs:
+      name:
+        type: string
+        required: true
+      python_version:
+        type: string
+        required: true
+      pip_version:
+        type: string
+        required: false
+        default: ">=23,<23.1"
+      poetry_version:
+        type: string
+        required: false
+        default: ">=1.4.0,<1.5"
+      artifact_name:
+        type: string
+      artifact_path:
+        type: string
+    secrets:
+      TECHONOMY_REPOS_GITHUB_TOKEN:
+        required: true
+
+jobs:
+  publish-python-package:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v3
+      - name: Download static files
+        uses: actions/download-artifact@v3
+        with:
+          name: ${{ inputs.artifact_name }}
+          path: ${{ inputs.artifact_path }}
```

Comment:
gaat dit ook werken als je geen artifacts hebt? (bijv een python-only package?)

Reply:
Een goede vraag! Voor nu is het verplicht maar moeten we wel optionaal gaan maken (pak ik als PR wel op!)